documentation/milestones/individual_milestones/static_site.md

```diff
@@ -0,0 +1,199 @@
+
+- Raises awareness about the Grants Equity project, its priorities, and ongoing workstreams
+- Establishes beta.grants.gov as the primary location that stakeholders can visit for project updates and previews of deliverables from each of the workstreams
+- Proves the successful completion of technical milestones that enable faster development without sacrificing code quality or security
+- Delivers an early win that both internal and external stakeholders can rally around, which helps build momentum and enthusiam for the project
+- Facilitates a parallel approach to development, in which new features can be built and tested on `beta.grants.gov` without risking or disrupting the existing functionality of legacy grants.gov
+
+### User stories
+
+- As a **full-time HHS staff member**, I want:
+  - the site to be accessible to members of the public and the Federal government, so we can use it to share information about the project with both internal and external stakeholders.
+  - the site to adopt modern branding and user interface (UI), so that stakeholders are excited to visit the page and can find the information they need more easily.
+- As a **grantor**, I want:
+  - to be able to access information about the NOFO Transformation project in a central location, so that I don't have to rely exclusively on email for updates about the project.
+  - the site to be user friendly and easy to navigate, so that I don't have to spend a lot of time looking for information that is relevant to me.
+- As a **prospective grant applicant**, I want:
+  - the site to be user friendly and easy to navigate, so that I don't have to spend a lot of time looking for information that is relevant to me.
+  - an opportunity to provide feedback or ask questions about the project, so that I can voice my concerns and help shape the direction of the project.
+- As **maintainer of the project** I want:
+  - most of the critical development infrastructure to be in place when we officially launch the site, so that we can deploy bug fixes or new features quickly once the site is live.
+- As an **open source contributor**, I want:
+  - the site to link to resources like the repository, chat, etc., so that I can easily learn where and how to participate in the project.
+
+## Technical description
+
+### Infrastructure Requirements
+
+The infrastructure developed to deploy and host the site should balance:
+
+- Code quality
+- Security
+- Delivery velocity
+- Cost & maintenance
+
+### User Experience Requirements
+
+The design and structure of the site should balance:
+
+- Usability
+- Accessibility
+- Site performance
+- Brand identity
+
+### Content Requirements
+
+Process for drafting and updating the content of the site should balance:
+
+- Speed & ease of content management
+- Need for review and approval
+
+### Definition of done
+
+- [ ] The following infrastructure requirements are satisfied:
+  - [ ] The code needed to build and deploy the site is merged to `main`
```

Comment:
Nit: we never actually say "the content is deployed to beta.grants.gov" as part of the definitions of done, from what I can tell. Let's make that explicit! Thanks.

Reply:
Good call, added it in [this commit](https://github.com/HHS/grants-equity/pull/243/commits/b1620b74ecc0d590e7fc06575fd8f47edc035bc6)